### ISSUE TITLE
[BUG] Process hangs if network resource unavailable

### ISSUE THREAD

**Issue by tgmct** [notepad-plus-plus/notepad-plus-plus#15540]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description of the Issue

Notepad++ hangs if the process is attempting to edit a file that resides on a network resource that is unavailable for some reason

### Steps To Reproduce

1.  Attempt to edit a previously opened file that resides on a network resource THAT IS DOWN.  For example: \\NAS1\volume1\home\stuff\workingonit.txt
2. NotePad++ hangs and becomes unresponsive.
3. Attempt to close NPP with the upper right corner "X".
4. "NotePad++ is not reponding" pop-up appears.
5.  "Close the Program" is selected in order to force close NPP.
6. Attempts to reopen NPP also hang with the same sequence.

### Current Behavior

see steps to repoduce

### Expected Behavior

some other sort of error message

### Debug Information

```shell
Unavailable - NPP is currently totally unresponsive until I either uninstall/reinstall or somehow get the network resource to return online
```

**Comment by xomx** [comment]
@tgmct 
Probably a duplicate of #8055 and similar.

I think if you try to wait long enough (depends on your system settings for the net-resource-connection-timeouts), N++ should be able to continue after such annoying delay. Some workarounds are in #12553 .

**Comment by donho** [comment]
@tgmct 
This issue might be fixed in Notepad++ v8.7:
https://community.notepad-plus-plus.org/topic/26105/notepad-v8-7-release-candidate

Could you guys try v8.7 and confirm the bug is fixed?

**Comment by donho** [comment]
Please try the latest state of fixed binary here (x64):
https://github.com/notepad-plus-plus/notepad-plus-plus/actions/runs/11118197845/artifacts/1998785683

The hanging issue should be improved considerably.
